pyuvdata/uvcal.py

```diff
@@ -432,6 +432,28 @@ def select(self, antenna_nums=None, antenna_names=None,
         if run_check:
             self.check(run_check_acceptability=run_check_acceptability)
 
+    def convert_to_gain(self):
+        if self.cal_type == 'gain':
+            raise ValueError('The data is already a gain cal_type.')
+        elif self.cal_type == 'delay':
+            phase_array = np.zeros((self.Nants_data, self.Nspws, self.Nfreqs, self.Ntimes, self.Njones))
+            for si in range(self.Nspws):
+                temp = -2 * np.pi * np.dot(self.delay_array[:, si, :, :, np.newaxis],
```

Comment:
I'm not sure how universal the sign convention in this exponent is. In other words, does a positive delay imply a certain direction of phase ramp, or is there ambiguity?
Should there be a delay_convention parameter to say whether the time*freq part has a plus or minus? Or an argument to this function which defaults to minus?